src/rdb.c

```diff
@@ -1385,6 +1385,7 @@ int rdbSaveBackground(char *filename, rdbSaveInfo *rsi) {
         int retval;
 
         /* Child */
+        setOOMScoreAdj(CONFIG_OOM_BGCHILD);
```

Comment:
you are missing this call in `rdbSaveToSlavesSockets`, and maybe for `RM_Fork`?
maybe a better place for this is inside `redisFork`?

Reply:
Thanks! it was on my todo list to check for other occurrences but slipped out.
I'm not sure about `RM_Fork()` which is why I also did not do that in `RedisFork()`, it may be very useful or very harmful depending on the specific, unknown context.